### PR TITLE
Jdc visitor update

### PR DESCRIPTION
**Pull request by jimguistical**
added update functionality to visitors

### CODE REVIEW

File: src/javascripts/helpers/data/visitorData/visitorsData.js [hunk 1]
```diff
@@ -32,6 +32,20 @@ const deleteVisitor = (firebaseKey) => new Promise((resolve, reject) => {
...
+// SINGLE VISITOR
+const getSingleVisitor = (firebaseKey) => new Promise((resolve, reject) => {
+  axios.get(`${dbUrl}/Visitors/${firebaseKey}.json`)
+    .then((response) => resolve(response.data))
+    .catch((error) => reject(error));
+});
+
+// UPDATE
+const updateVisitor = (firebaseKey, visitorObject) => new Promise((resolve, reject) => {
+  axios.patch(`${dbUrl}/Visitors/${firebaseKey}.json`, visitorObject)
```
Comment (ajawashington) [inline src/javascripts/helpers/data/visitorData/visitorsData.js:44]: I know your firebase is already set up so don't change anything, but it is good practice to have all your collections to be lowercase.

**Review by ajawashington — COMMENTED** [review]
Just left a quick note. 🗒️

File: src/javascripts/helpers/data/visitorData/visitorsData.js (hunk 1 above)
Comment (jimguistical) [reply src/javascripts/helpers/data/visitorData/visitorsData.js:44]: Okay, thanks for the head's up - will remember that going forward!

**Review by fimoefive — APPROVED** [review]
Good Job, on reaching CRUD and MVP man

**Review by Mitchell-Crumbley — APPROVED** [review]

**Pull request merged**